tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/webapp/BackupServiceElasticSearchTest.java

```diff
@@ -1,821 +1,834 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
```

Comment:
This is temporarily commented as I'm checking if all the tests  cases have been ported already